pkg/operator2/operator.go

```diff
@@ -557,7 +558,7 @@ func (c *authOperator) getAPIServerIPs() ([]string, error) {
 
 		ips := make([]string, 0, len(subset.Addresses))
 		for _, address := range subset.Addresses {
-			ips = append(ips, fmt.Sprintf("%s:%d", address.IP, targetPort))
```

Comment:
why the int -> int conversion?

Reply:
You're right, not needed.

I either copied this or did it out of habit.  In lots of other places where this change is needed, the port is an `int64`, not `int`.